src/subcommand/server.rs

```diff
@@ -283,6 +282,7 @@ impl Server {
         )
         .route("/r/inscription/{inscription_id}", get(r::inscription))
         .route("/r/metadata/{inscription_id}", get(r::metadata))
+        .route("/r/sat/{sat_number}/at/{index}", get(r::sat_at_index))
```

Comment:
I think this should be moved back, so it's sorted.

Reply:
We try to keep these sorted, so `r/sat/{sat_number}/at/{index}/content` should be moved up.

Reply:
I always just do `:sort` in vim and use whatever order that uses.

Reply:
Ah, sorry, I wasn't looking at the whole diff and didn't see that proxiable_routes had been moved into a separate section.